### PR TITLE
Ltd 5000 add country highlighting

### PR DESCRIPTION
**Pull request by markj0hnst0n**
### Aim

Highlight the country on denials search page

Corresponding API pr here: https://github.com/uktrade/lite-api/pull/1982

[LTD-5000](https://uktrade.atlassian.net/browse/LTD-5000)

[LTD-5000]: https://uktrade.atlassian.net/browse/LTD-5000?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: caseworker/cases/views/main.py [hunk 1]
```diff
@@ -698,6 +698,9 @@ def get_context_data(self, **kwargs):
         if search:
             response = search_denials(request=self.request, search=search, filter=filter).json()
             results = response["results"]
+            for result in results:
+                if result["country"]:
+                    result["country"] = f"<mark>{result['country']}</mark>"
```
Comment (depsiatwal) [inline caseworker/cases/views/main.py:703]: As marking is happening on BE it makes sense to perhaps contain this logic there. 

I would check the 'filter' and if it matches the country returned then add the markup

Comment (markj0hnst0n) [reply caseworker/cases/views/main.py:703]: I think it's _technically_ possible to highlight filter fields in the backend but not very convenient with `django_elasticsearch_dsl_drf` it requires manually re-writing the search queries and I'm not even sure how to get under the hood to do that.

From what I understand highlighting is used for highlighting search terms only usually not filter terms.

Comment (depsiatwal) [reply caseworker/cases/views/main.py:703]: yeah I was thinking more from a consistent stand point. Marking is then done all in one place. 
any how I think we should still definitely check the 'filter' query matches the country field.

Comment (depsiatwal) [reply caseworker/cases/views/main.py:703]: yeah I think you would need to write something on the get or filter if it's too much of a pain 
also happy for it to go here for first iteration.

**Comment by markj0hnst0n** [comment]
closed in favour of back end change

**Pull request closed without merging**